File: cypress/cypress/integration/cypress_poc/compra.spec.js
import { login } from "./pages/login.page"
import { AllItens } from "./pages/all_itens.page"
import { Informacoes } from "./pages/your_information.page"
import { ResumoCompra } from "./pages/overview.page"

describe('Comprar', () => {

  beforeEach(() => {
    cy.visit('https://www.saucedemo.com/')
    login('standard_user', 'secret_sauce')
  })

  it('Fazer uma compra com sucesso', () => {
    AllItens.comprar()
    Informacoes.preencher_infos()
    ResumoCompra.confirmar()
    cy.get('.complete-header')
  })

})

File: selenium/tests/compra.spec.rb
require_relative '../pages/login.page'
require_relative '../pages/all_itens.page'
require_relative '../pages/your_information.page'
require_relative '../pages/overview.page'

RSpec.describe 'Comprar' do

  before(:example) do
    $driver.get 'https://www.saucedemo.com/'
    login('standard_user', 'secret_sauce')
  end

  it 'Fazer uma compra com sucesso' do
    AllItens.comprar
    Informacoes.preencher_infos
    ResumoCompra.confirmar
    $driver.find_element(:class, 'complete-header')
  end

end
